chinese/templates/chinese_new.py

```diff
@@ -10,7 +10,7 @@
 
 import re
 from anki import hooks
-from anki.utils import stripHTML
+from anki.utils import strip_html as stripHTML
```

Comment:
Good work on tackling the deprecations!

I suggest here to use the default provided by the ankitects and change the function calls inside the code accordingly.
It's best not to confuse and another dev might think: Why, there is a deprecation let's fix it... ah, they just kept the old name, figures.

By the way. Could it be that @luoliyan or @joeminicucci need more collaborators to check the PRs and all? I would volunteer having been using this addon for a long time and also am collaborating to other Anki addons (namely https://github.com/zjosua/anki-mc/issues, which is much simpler admittedly 🙂).